locale/fi/LC_MESSAGES/django.po

```diff
@@ -17,391 +17,511 @@ msgstr ""
-msgstr "RESPA Resurssien varausjärjestelmä"
+#: resources/admin/period_inline.py:38
+msgid "HH:mm"
+msgstr ""
 
-#: resources/admin.py:18
-msgid "RESPA Administration"
-msgstr "RESPA Ylläpito"
+#: resources/api/reservation.py:53
+msgid "Maximum number of active reservations for this resource exceeded."
+msgstr "Liikaa voimassa olevia varauksia tähän resurssiin."
 
-#: resources/apps.py:6
+#: resources/apps.py:7
 msgid "Resource app"
 msgstr "Resurssisovellus"
 
-#: resources/models.py:76
-msgid "Time of creation"
-msgstr "Lisäysaika"
+#: resources/models/availability.py:14
+msgid "open"
+msgstr "auki"
 
-#: resources/models.py:77
-msgid "Created by"
-msgstr "Lisääjä"
+#: resources/models/availability.py:14
+msgid "closed"
+msgstr "kiinni"
 
-#: resources/models.py:79
-msgid "Time of modification"
-msgstr "Muokkausaika"
+#: resources/models/availability.py:105
+#, fuzzy
+#| msgid "Exceptional period"
+msgid "exception parent period"
+msgstr "Poikkeusaikaväli"
 
-#: resources/models.py:80
-msgid "Modified by"
-msgstr "Muokkaaja"
+#: resources/models/availability.py:106
+msgid "Exceptional period"
+msgstr "Poikkeusaikaväli"
+
+#: resources/models/availability.py:107 resources/models/reservation.py:18
+#: resources/models/resource.py:307
+msgid "Resource"
+msgstr "Resurssi"
+
+#: resources/models/availability.py:109 resources/models/resource.py:71
+#: resources/models/unit.py:63
+msgid "Unit"
+msgstr "Toimipiste"
+
+#: resources/models/availability.py:112
+msgid "Start date"
+msgstr "Alkupäivä"
+
+#: resources/models/availability.py:113
+msgid "End date"
+msgstr "Loppupäivä"
+
+#: resources/models/availability.py:114
+msgid "Length of period"
+msgstr "Aikavälin kesto"
 
-#: resources/models.py:172 resources/models.py:234 resources/models.py:255
-#: resources/models.py:276 resources/models.py:584
+#: resources/models/availability.py:117 resources/models/equipment.py:11
+#: resources/models/equipment.py:23 resources/models/equipment.py:36
+#: resources/models/resource.py:33 resources/models/resource.py:54
+#: resources/models/resource.py:75 resources/models/unit.py:16
 msgid "Name"
 msgstr "Nimi"
 
-#: resources/models.py:173 resources/models.py:277 resources/models.py:585
+#: resources/models/availability.py:118 resources/models/resource.py:76
+#: resources/models/unit.py:17
 msgid "Description"
 msgstr "Kuvaus"
 
-#: resources/models.py:175 resources/models.py:287
-msgid "Location"
-msgstr "Sijainti"
+#: resources/models/availability.py:120 resources/models/availability.py:271
+msgid "Closed"
+msgstr "Kiinni"
 
-#: resources/models.py:176
-msgid "Time zone"
-msgstr "Aikavyöhyke"
+#: resources/models/availability.py:123
+msgid "period"
+msgstr "aikaväli"
 
-#: resources/models.py:181
-msgid "Street address"
-msgstr "Katuosoite"
+#: resources/models/availability.py:124
+msgid "periods"
+msgstr "aikavälit"
 
-#: resources/models.py:182
-msgid "Postal code"
-msgstr "Postinumero"
+#: resources/models/availability.py:210
+#, fuzzy
```

Comment:
leftover fuzzy here? The strings below seem very much alike, can they be combined?